client/incus_instances.go

```diff
@@ -2528,6 +2528,10 @@ func (r *ProtocolIncus) ConsoleInstance(instanceName string, console api.Instanc
 		return nil, fmt.Errorf("The server is missing the required \"console_vga_type\" API extension")
 	}
 
+	if console.Force && !r.HasExtension("console_force") {
+		return nil, fmt.Errorf("The server is missing the required \"console_force\" API extension")
```

Comment:
I would write this with back ticks to avoid the need for escaping:
```suggestion
		return nil, fmt.Errorf(`The server is missing the required "console_force" API extension`)
```

Reply:
Done.